src/extension.ts

```diff
@@ -44,6 +44,7 @@ const languages: {
   sparql: { module: "tree-sitter-sparql" },
   starlark: { module: "tree-sitter-python" },
   talon: { module: "tree-sitter-talon" },
+  terraform: { module: "tree-sitter-hcl" },
```

Comment:
I'm actually a bit surprised this works, given your Makefile. Shouldn't this be `tree-sitter-hcl`? That being said, if you implement my other suggestions, you should leave this line alone. Just a bit surprised it works as is